tests/renderer/binary-spec.ts

```diff
@@ -262,66 +233,22 @@ describe('binary', () => {
       downloadResolve();
     });
 
-    it(`does not download a version again`, async () => {
-      (fs.existsSync as jest.Mock).mockReturnValue(true);
-
-      const version = '3.0.0';
-      mockState.versions[version] = {
-        source: VersionSource.remote,
-        state: VersionState.unknown,
-        version,
-      };
-
-      await setupBinary(mockState, version);
-
-      expect(electronDownload).not.toHaveBeenCalled();
-      expect(mockState.versions[version].state).toBe('ready');
-    });
-
```

Comment:
Removed because this is effectively the same as `returns the same promise if called twice for the same version`